torch/_jit_internal.py

```diff
@@ -165,13 +165,16 @@ def ignore(fn):
     return fn
 
 
-def _parameter_list(fn):
+def _parameter_list(parameter_names_fn):
     """
     Decorator to denote that a function returns a list of all the parameters
     in a module
     """
-    fn._is_parameter_list = True
-    return fn
+    def decorator(fn):
```

Comment:
Use functools wrap so that the docs & such remain n?

Reply:
It returns the function itself so those should all be preserved already